app/controllers/playlists_controller.rb

```diff
@@ -0,0 +1,60 @@
+class PlaylistsController < ApplicationController
+
+  before_filter :find_playlist, only: [:show, :edit, :update, :destroy]
+  before_filter :find_user, only: [:new, :create]
```

Comment:
I think we need it for all methods as far show expects to receive user's and playlist's id to represent correct value